client/packages/admin/src/index.ts

```diff
@@ -926,4 +928,6 @@ export {
   type ValueTypes,
   type InstantSchemaDef,
   type InstantUnknownSchema,
+  type InstaQLEntity,
+  type InstaQLResult,
```

Comment:
Unrelated to this PR, but I noticed we forgot to export these in admin. Thought it wasn't worth an extra PR